### PR TITLE
feat: added last_used in login page

### LINKED ISSUE #707: [🕹️] "Last Used" indicator on login page
### What side quest or challenge are you solving?

Users get confused how they signed up last time

### Points

300 Points

### Description

Add a subtle "Last used" indicator

375053241-bccfc390-6baa-47da-a417-128d1b27e319

Examples:
- https://x.com/dannypostmaa/status/1810864175503339774
- https://x.com/rauchg/status/1816546405135376595
- https://x.com/nbashaw/status/1819039902488842552
- https://x.com/steventey/status/1820517962954178694
- https://x.com/hanko_io/status/1844038475659743263

### Provide proof that you've completed the task

Open a PR with a preview and description of what you have done

### PR DESCRIPTION
**Pull request by RajuGangitla**
I added a last used text on login form and im using local storage the value 

Fixes #707 

[screen-capture (22).webm](https://github.com/user-attachments/assets/f2849799-9154-46cc-a84d-a16d7ef5245a)

### CODE REVIEW

**Comment by RajuGangitla** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by Meetpidev** [comment]
> I have read the CLA Document and I hereby sign the CLA
Hey Please tell me How to sign CLA ? i dont found any link for it in CLA.md

**Comment by RajuGangitla** [comment]
Just comment that

File: components/hooks/useLastUsed.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    }, [lastUsed]);
+
+    return [lastUsed, setLastUsed] as const;
+}
+
+export const LastUsed = ({ className }: { className?: string | undefined }) => {
+    return (
+        <span className={classNames("text-gray-500 absolute right-1 text-xs", className ?? "")}>{("last_used")}</span>
+    );
+};
```
Comment (mfts) [inline components/hooks/useLastUsed.tsx]: can we give the last used indicator a little more love :)

**Review by mfts — CHANGES_REQUESTED** [review]
need more love on the last_used indicator. 

perhaps something like this
CleanShot 2024-10-12 at 00 38 24@2x

File: components/hooks/useLastUsed.tsx (hunk 1 above)
Comment (RajuGangitla) [reply components/hooks/useLastUsed.tsx]: Like a tool tip @mfts

Comment (Meetpidev) [reply components/hooks/useLastUsed.tsx]: @mfts  we can make the UI better than present..  /assign

Comment (RajuGangitla) [reply components/hooks/useLastUsed.tsx]: I will improve it

Comment (Meetpidev) [reply components/hooks/useLastUsed.tsx]: /assign

Comment (RajuGangitla) [reply components/hooks/useLastUsed.tsx]: @Meetpidev I'm already working on this issue

**Comment by RajuGangitla** [comment]
image

@mfts improved the last used

**Comment by mfts** [comment]
@RajuGangitla thanks a ton. last request, because we had a contribution on the sign up just earlier than you. 

could you please merge in the current main branch and resolve any conflicts. 

btw the design looks 🔥  now

**Comment by RajuGangitla** [comment]
@mfts will do it one minute please wait

**Comment by RajuGangitla** [comment]
@mfts done merged main branch

**Comment by mfts** [comment]
@RajuGangitla you were faster than me 💪

I just merged in the login change to main. There's just one conflict left. Should be quick to fix. 

Thanks again :)

**Comment by RajuGangitla** [comment]
@mfts resolved

**Review by mfts — APPROVED** [review]
Legend

giphy-5

**Comment by mfts** [comment]
@RajuGangitla i guess not quite yet. 

build error:
```
9:13  Error: Definition for rule '@calcom/eslint/avoid-web-storage' was not found.  @calcom/eslint/avoid-web-storage
19:13  Error: Definition for rule '@calcom/eslint/avoid-web-storage' was not found.  @calcom/eslint/avoid-web-storage
30:13  Error: Definition for rule '@calcom/eslint/avoid-web-storage' was not found.  @calcom/eslint/avoid-web-storage
```

**Comment by RajuGangitla** [comment]
@mfts resolved that running the command npm run build .....

File: lib/webstorage.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
+/**
+ * Provides a wrapper around localStorage(and sessionStorage(TODO when needed)) to avoid errors in case of restricted storage access.
+ *
+ * TODO: In case of an embed if localStorage is not available(third party), use localStorage of parent(first party) that contains the iframe.
+ */
+export const localStorage = {
+    getItem(key: string) {
+        try {
+            // eslint-disable-next-line @calcom/eslint/avoid-web-storage
```
Comment (mfts) [inline lib/webstorage.ts]:
```suggestion
            // eslint-disable-next-line
```

File: lib/webstorage.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+            return window.localStorage.getItem(key);
+        } catch (e) {
+            // In case storage is restricted. Possible reasons
+            // 1. Third Party Context in Chrome Incognito mode.
+            return null;
+        }
+    },
+    setItem(key: string, value: string) {
+        try {
+            // eslint-disable-next-line @calcom/eslint/avoid-web-storage
```
Comment (mfts) [inline lib/webstorage.ts]:
```suggestion
            // eslint-disable-next-line
```

File: lib/webstorage.ts [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+        } catch (e) {
+            // In case storage is restricted. Possible reasons
+            // 1. Third Party Context in Chrome Incognito mode.
+            // 2. Storage limit reached
+            return;
+        }
+    },
+    removeItem: (key: string) => {
+        try {
+            // eslint-disable-next-line @calcom/eslint/avoid-web-storage
```
Comment (mfts) [inline lib/webstorage.ts]:
```suggestion
            // eslint-disable-next-line
```

**Review by mfts — COMMENTED** [review]
i'll just remove these

File: lib/webstorage.ts (hunk 4 above)
Comment (RajuGangitla) [reply lib/webstorage.ts]: thanks for resolving i already resolved and about to push my code 

image

**Pull request merged**